db/models.py

```diff
@@ -14,3 +14,43 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField(blank=True)
+    actors = models.ManyToManyField(Actor, related_name="actor")
```

Comment:
The `related_name` for the `actors` field in the `Movie` model is set to 'actor'. This might cause confusion as it suggests a singular relationship. Consider using a plural form like 'movies' to indicate the reverse relation from `Actor` to `Movie`.